src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -277,8 +274,10 @@ internal partial class CodeFixService : ICodeFixService
                 FixAllCodeActionContext fixAllContext = null;
                 if (fixAllProviderInfo != null)
                 {
-                    var codeFixProvider = (fixer as CodeFixProvider) ?? new WrapperCodeFixProvider((ISuppressionFixProvider)fixer, diagnostics);
-                    fixAllContext = FixAllCodeActionContext.Create(document, fixAllProviderInfo, codeFixProvider, diagnostics, this.GetDocumentDiagnosticsAsync, this.GetProjectDiagnosticsAsync, cancellationToken);
+                    var codeFixProvider = (fixer as CodeFixProvider) ?? new WrapperCodeFixProvider((ISuppressionFixProvider)fixer, diagnostics.Select(d => d.Id).Distinct().ToImmutableArray());
```

Comment:
Why did we move `.Select(d => d.Id).Distinct().` invocation out of the WrapperCodeFixProvider ctor? All the callers now need to invoke this right?